src/components/icons/router.icons.ts

```diff
@@ -1,7 +1,9 @@
 import express from 'express'
+import * as controller from './controller.icons'
 const router = express.Router()
 
 const prefix = '/icons'
+router.get('/newReleaseGitlab', controller.newRelease)
```

Comment:
can we update this route as `.post` since its the endpoint that will be created after

Reply:
Resolved.